pkg/controllers/syncer/translator/namespaced_translator.go

```diff
@@ -80,6 +80,10 @@ func (n *namespacedTranslator) SyncToHostCreate(ctx *context.SyncContext, vObj,
 			ctx.Log.Debugf("error syncing %s %s/%s to physical cluster: %v", n.name, vObj.GetNamespace(), vObj.GetName(), err)
 			return ctrl.Result{RequeueAfter: time.Second}, nil
 		}
+		if kerrors.IsAlreadyExists(err) {
+			ctx.Log.Debugf("ignoring syncing %s %s/%s to physical cluster as it already exists", n.name, vObj.GetNamespace(), vObj.GetName())
```

Comment:
Beyond a need to update this PR again with the new generics code, I'll have to give this some thought.  It has essentially moved to [here](https://github.com/loft-sh/vcluster/blob/main/pkg/syncer/syncer.go#L546) but placing it in such a central place means that any resource trying to create on sync that already exists could take over the original object and start making changes to it.  It might be better to let it fail here as its already doing, or put checks on specific resources like the [service accounts](https://github.com/loft-sh/vcluster/blob/main/pkg/controllers/resources/serviceaccounts/syncer.go#L56).